### PR TITLE
Update pre-requisites for .NET 4.6.2

### PR DESCRIPTION
**Pull request by jdom**

### CODE REVIEW

**Comment by mikekinsman** [comment]
Docs Build status updates of commit _[231af5a](https://github.com/jdom/portaldocs/pull/242/commits/231af5a5b4d5f6c26434445c32310cd8d7a94f66)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-7cb8a4a4)

<a id="7cb8a4a4"></a>
###
 - **[Error]** `````Publish service's config file: .openpublishing.publish.config.json is not found. Exception: `````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C12%5C3d86255f-f488-188e-02a3-9e92aa67830d%5CPullRequest%5C201907121358349531-242%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=zA0m%2FU6vnfReZeFsSiEMx441vl6FqhrampXAgzFHK3M%3D&st=2019-07-12T13%3A53%3A49Z&se=2019-08-12T13%3A58%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mikekinsman** [comment]
Docs Build status updates of commit _[b57cf21](https://github.com/jdom/portaldocs/pull/242/commits/b57cf21a915cf0971a710287d4ce6e69527e0068)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-59ff3922)

<a id="59ff3922"></a>
###
 - **[Error]** `````Publish service's config file: .openpublishing.publish.config.json is not found. Exception: `````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C12%5C3d86255f-f488-188e-02a3-9e92aa67830d%5CPullRequest%5C201907121402556066-242%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=jHBSavtyZ8XyfwJnVGDMegLP7wGdlTdAYeN%2FwnuCTqA%3D&st=2019-07-12T13%3A58%3A10Z&se=2019-08-12T14%3A03%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mikekinsman** [comment]
Docs Build status updates of commit _[c9da62f](https://github.com/jdom/portaldocs/pull/242/commits/c9da62f522b789ac94c9e846d4b2bc9ccc1c6228)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-3f3cec6d)

<a id="3f3cec6d"></a>
###
 - **[Error]** `````Publish service's config file: .openpublishing.publish.config.json is not found. Exception: `````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C12%5C3d86255f-f488-188e-02a3-9e92aa67830d%5CPullRequest%5C201907121418133457-242%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ndfPpHFHT6pu6zgmrZMCuIR7QjS0q%2FE%2FLNx9cFTyxkI%3D&st=2019-07-12T14%3A13%3A27Z&se=2019-08-12T14%3A18%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: portal-sdk/templates/top-extensions-install-software.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ Install the following software. Your team should be aware of the most current do
 
 ### Visual Studio Code (or other similar IDE)
 
-1. Install [.NET 4.5.2](https://www.microsoft.com/en-us/download/details.aspx?id=42642)
+1. Install [.NET Framework 4.6.2 Developer Pack](http://go.microsoft.com/fwlink/?LinkId=780617)
 
-1. Install [Build Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159)
+1. Install [Build Tools for Visual Studio 2017](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=15)
```
Comment (nickharris) [inline portal-sdk/templates/top-extensions-install-software.md]: looks like the new 2017 install is not offline, can you screenshot the settings. I think you will want these 
image

also can these two steps be combined into one now i.e you can select the 4.6.2 options as per image below

image

does the new installer already put msbuild on the path?

File: portal-sdk/templates/top-extensions-install-software.md [hunk 2]
```diff
@@ -28,11 +28,11 @@ Install the following software. Your team should be aware of the most current do
...
 
-      * Add it to your environmental path by running the following command from an administrative command prompt `setx PATH "%PATH%;C:\Program Files (x86)\MSBuild\14.0\Bin\"`
+      * Add it to your environmental path by running the following command from an administrative command prompt `setx PATH "%PATH%;%ProgramFiles(x86)%\Microsoft Visual Studio\2017\BuildTools\MSBuild\Current\Bin"`
```
Comment (nickharris) [inline portal-sdk/templates/top-extensions-install-software.md]: msbuild on clean VM was dropped under %ProgramFiles(x86)%\Microsoft Visual Studio\2017\BuildTools\MSBuild\Bin

i.e no \Current folder.  if your seeing that on yours, maybe we should add both to this command.

**Pull request closed without merging**